build.gradle.kts

```diff
@@ -63,12 +63,11 @@ buildscript {
 plugins {
     `java-library`
     idea
-    id(protobufPlugin)
-    id(errorPronePlugin)
+    protobuf
+    errorprone
     kotlin("jvm")
-    `force-jacoco`
     `detekt-code-analysis`
```

Comment:
It doesn't have to be applied to the root project, unless a root project is a module itself.